### ISSUE TITLE
Loader issue with package contain .vue file

### ISSUE THREAD

**Issue by adrian999999** [JeffreyWay/laravel-mix#897]
```
 error  in ./node_modules/vue2-google-maps/dist/components/streetViewPanorama.vue

Module parse failed: /Users/adrian/apps/highschooltutors/node_modules/vue2-google-maps/dist/components/streetViewPanorama.vue Unexpected token (1:0)
You may need an appropriate loader to handle this file type.
<template>
   <div class="vue-street-view-pano-container">
     <div ref="vue-street-view-pano" class="vue-street-view-pano"></div>```

 @ ./node_modules/vue2-google-maps/dist/main.js 47:26-72
 @ ./resources/assets/js/app.js
 @ multi ./resources/assets/js/app.js ./resources/assets/sass/app.scss

**Comment by JeffreyWay** [comment]
Mix won't process any .vue files in your node_modules directory right now. It ignores that directory entirely.

**Comment by ankurk91** [comment]
@JeffreyWay 
Sometime we may need to process ``.vue`` coming from ``node_modules``, 

There are npm packages, for example - 

https://github.com/ankurk91/vue-flatpickr-component
https://www.npmjs.com/package/vue-nprogress

There may be more

FYI: ``.vue`` files are being processed in v0.12.1 and if not in v1.0.0 then it is a breaking change,

**Comment by msonowal** [comment]
same here it is failing for my installed vue components packages
@JeffreyWay what is the fix for this?

**Comment by bbashy** [comment]
Example of breaking change;

```
 error  in ./node_modules/vue-full-calendar/components/FullCalendar.vue

Module parse failed: /Users/ben/path/to/project/node_modules/vue-full-calendar/components/FullCalendar.vue Unexpected token (1:0)
You may need an appropriate loader to handle this file type.
| <template>
|     <div ref="calendar" id="calendar"></div>
| </template>

 @ ./node_modules/vue-full-calendar/index.js 1:19-59
```

**Comment by JeffreyWay** [comment]
Okay I'll get this fixed right now.

**Comment by JeffreyWay** [comment]
Fixed in 1.0.3.
